tsconfig.json

```
@@ -4,9 +4,9 @@
     "outDir": "lib"
   },
   "include": [
-    "src", "typings"
+    "src"
   ],
   "exclude": [
-    "**/*.spec.ts", "**/*.spec.tsx"
+    "**/*.spec.ts", "**/*.spec.tsx", "src/util/test-helper.ts"
```

No problem! Here it was me giving a wrong suggestion! Sorry for that.
```suggestion
    "**/*.spec.ts", "**/*.spec.tsx", "src/utils/test-helper.ts"
```